src/timelines/main.js

```diff
@@ -1,53 +1,43 @@
-import preamble from "./preamble";
-import taskBlock from "./taskBlock";
-import { countdown } from "@brown-ccv/behavioral-task-trials";
+import preamble from "./preamble"
+import taskBlock from "./taskBlock"
+import { countdown } from "@brown-ccv/behavioral-task-trials"
 import { cameraStart, cameraEnd } from "../trials/camera"
-import { lang, config } from "../config/main";
-import { practiceBlock } from "../config/practice";
-import { tutorialBlock } from "../config/tutorial";
-import { exptBlock1, exptBlock2 } from "../config/experiment";
-import { showMessage } from "@brown-ccv/behavioral-task-trials";
+import { lang, envConfig } from "../config/main"
+import { showMessage } from "@brown-ccv/behavioral-task-trials"
 import {
   ageCheck,
   sliderCheck,
   demographics,
   iusSurvey,
   debrief,
-} from "../trials/quizTrials";
+} from "../trials/quizTrials"
 
-let primaryTimeline = [
-  preamble,
-  ageCheck,
-  sliderCheck,
-  countdown({ message: lang.countdown.message1 }),
-  taskBlock(practiceBlock),
-  countdown({ message: lang.countdown.message2 }),
-  taskBlock(exptBlock1),
-  demographics,
-  iusSurvey,
-  debrief,
-];
+const tl = (experimentConfig) => {
+  let timeline = [preamble]
 
-if (config.USE_CAMERA) {
-  primaryTimeline.splice(1,0,cameraStart())
-  primaryTimeline.push(cameraEnd(5000))
-}
+  if (!envConfig.USE_MTURK) timeline.push(ageCheck, sliderCheck)
 
-primaryTimeline.push(showMessage(config, {
-  duration: 5000,
-  message: lang.task.end,
-}))
-
-const mturkTimeline = [
-  preamble,
-  countdown({ message: lang.countdown.message1 }),
-  taskBlock(tutorialBlock),
-  countdown({ message: lang.countdown.message2 }),
-  taskBlock(exptBlock2),
-  showMessage(config, {
+  timeline.push(countdown({ message: lang.countdown.message1 }))
+
+  const block = envConfig.USE_MTURK ? experimentConfig.tutorialBlock : experimentConfig.practiceBlock
+  const block2 = envConfig.USE_MTURK ? experimentConfig.exptBlock2 : experimentConfig.exptBlock1
```

Comment:
I'd minimize the mturk logic, have one place that shows conditional pushing, but then otherwise keep the timeline simple